extension-spring/src/main/java/io/smallrye/openapi/spring/SpringParameter.java

```diff
@@ -16,7 +16,9 @@ public enum SpringParameter {
     MATRIX_PARAM(SpringConstants.MATRIX_PARAM, Parameter.In.PATH, Parameter.Style.MATRIX, Parameter.Style.MATRIX),
     QUERY_PARAM(SpringConstants.QUERY_PARAM, Parameter.In.QUERY, null, Parameter.Style.FORM),
     HEADER_PARAM(SpringConstants.HEADER_PARAM, Parameter.In.HEADER, null, Parameter.Style.SIMPLE),
-    COOKIE_PARAM(SpringConstants.COOKIE_PARAM, Parameter.In.COOKIE, null, Parameter.Style.FORM);
+    COOKIE_PARAM(SpringConstants.COOKIE_PARAM, Parameter.In.COOKIE, null, Parameter.Style.FORM),
+    PARAMETER_OBJECT(SpringConstants.PARAMETER_OBJECT, null, null, null),
+    JAKARTA_QUERY_PARAM(SpringConstants.JAKARTA_QUERY_PARAM, Parameter.In.QUERY, null, Parameter.Style.FORM);
```

Comment:
Why is this needed ?

Reply:
@see https://github.com/smallrye/smallrye-open-api/pull/1823#discussion_r1591279541